rotkehlchen/tests/unit/test_ethereum_airdrops.py

```diff
@@ -135,6 +135,22 @@
             'decimals': 18,
         },
     },
+    'eigen': {
+        'asset_identifier': 'EIGEN_TOKEN_PRE_RELEASE',
+        'url': 'eigenfoundation.org',
+        'api_url': 'https://claims.eigenfoundation.org/clique-eigenlayer-api/campaign/eigenlayer/credentials?walletAddress={address}',
```

Comment:
should we have some sanity nighly check for these API's response? I'm not sure but they can change it's schema or path without any versioning. So thinking what can go wrong if they change anything..

Reply:
This is a good idea. Can be done in another PR and should be left as a note to not forget it.